data/transactions/transaction.go

```diff
@@ -366,6 +366,9 @@ func (tx Transaction) RelevantAddrs(spec SpecialAddresses, proto config.Consensu
 		if tx.AssetTransferTxnFields.AssetCloseTo != (basics.Address{}) {
 			addrs = append(addrs, tx.AssetTransferTxnFields.AssetCloseTo)
 		}
+		if tx.AssetTransferTxnFields.AssetSender != (basics.Address{}) {
+			addrs = append(addrs, tx.AssetTransferTxnFields.AssetSender)
+		}
```

Comment:
looks like an unrelated bug fix to me.. ( not opposing, but would rather seeing in a separate PR )

Reply:
Pulled these out into a new PR: https://github.com/algorand/go-algorand/pull/948